### ISSUE TITLE
search.get_next_page() does not work

### ISSUE THREAD

**Issue by Lissanro** [salvoventura/pyunsplash#13]
This code fails to work, am I doing something wrong? This is the minimum code that triggers the issue. I'm trying to get next page in search results.

```
from pyunsplash import PyUnsplash
pu = PyUnsplash(api_key="api_key")
search = pu.search(type_="photos", query="bear")
if search.has_next:
    search.get_next_page()
```

```
TypeError                                 Traceback (most recent call last)

<ipython-input-16-b0e2b3485380> in <module>()
      2 search = pu.search(type_="photos", query="bear")
      3 if search.has_next:
----> 4     search.get_next_page()

/usr/local/lib/python3.6/dist-packages/pyunsplash/src/unpage.py in get_next_page(self)
    118 
    119     def get_next_page(self):
--> 120         return self.__class__(url=self.link_next, api_key=self.api_key)
    121 
    122     def get_previous_page(self):

TypeError: __init__() missing 1 required positional argument: 'where'
```

**Comment by salvoventura** [comment]
Hello Lissandro,
 thanks for reporting this. I was able to reproduce and confirm the issue. I have a fix that I will push soon.
 You code is correct, however keep in mind that currently, if you want to keep looping the next page, you should be doing as follows:

    search = py_un.search(type_="photos", query="bear")
    while search.has_next:
        print(search.link_next, search.link_last)
        search = search.get_next_page()  # <-- Note that get_next_page returns an object

I will also add an example for search, which was missing.

Salvatore

**Comment by salvoventura** [comment]
This has been resolved in the latest release, which is 1.0.0rc2, available on PyPI.

Thanks for reporting!
